misc/toml-parser/src/main/java/io/ballerina/toml/semantic/ast/TomlTransformer.java

```diff
@@ -457,4 +490,32 @@ public TomlNode transform(BoolLiteralNode boolLiteralNode) {
     public TomlNode transform(IdentifierLiteralNode identifierLiteralNode) {
         return new TomlUnquotedKeyNode(identifierLiteralNode.value().text(), getPosition(identifierLiteralNode));
     }
+
+    /**
+     * Gives Location of list of TomlNodes
+     * Should only call this method when the execution order is ensured.
+     *
+     * @param keys AST TomlNode key list
+     * @return TomlNodeLocation
+     */
+    private TomlNodeLocation getTomlNodeListLocation(List<TomlNode> keys) {
```

Comment:
```suggestion
    private TomlNodeLocation getTomlNodeListLocation(List<? extends TomlNode> TomlNodeList) {
```
Can use upper bounded wildcard to generalize. 
Also, shall we use a more generic name instead of "keys". Since the method can be used to get the location of any kind of TomlNode list.